contracts/service-registry/src/state.rs

```diff
@@ -173,9 +210,10 @@ pub fn deregister_chains_support(
     verifier: VerifierAddress,
 ) -> Result<(), ContractError> {
     for chain in chains {
-        VERIFIERS_PER_CHAIN.remove(storage, (&service_name, &chain, &verifier));
+        VERIFIERS_PER_CHAIN
+            .remove(storage, (service_name.clone(), chain, verifier.clone()))
+            .map_err(|_| ContractError::DeregistrationFailed)?;
```

Comment:
why change the error here? None of the other function have that

Reply:
removed.